src/conf_file.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2009-2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_CONF_FILE_H
+#define BITCOIN_CONF_FILE_H
+
+static const char* DEFAULT_BITCOIN_CONF_TEXT = R"(##
+## bitcoin.conf configuration file
+##
+
+# [network]
+# Add a node IP address to connect to and attempt to keep the connection open. This option can be set multiple times.
+#addnode=randomchars.onion
+# Bind to given address and always listen on it. (default: 0.0.0.0). Use [host]:port notation for IPv6. Append =onion to tag any incoming connections to that address and port as incoming Tor connections
+#bind=127.0.0.1:8333
```

Comment:
The default behaviour is to listen on 0.0.0.0; to listen on all interfaces.

In my opinion this option doesn't need to be mentioned here - if the aim is to include "basic things that a new user could understand", the default behaviour will suit most users, whereas the implicit suggestion to bind to localhost thereby explicitly preventing remote connections, seems harmful.

Reply:
> I was referring to L16 #bind=127.0.0.1:833

Its an example for using `bind`

Reply:
You mention the default binding is 0.0.0.0; which I understand to mean the server will listen on all network interfaces.

And we know listening on 127.0.0.1 would instruct the server not to listen on any interface; not to receive remote connections, but even a technical user might not realize this.

Maybe swapping the default with the example would reduce user error?

```suggestion
 # Bind to given address and always listen on it (eg 192.249.249.1). Use [host]:port notation for IPv6. Append =onion to tag any incoming connections to that address and port as incoming Tor connections 
#bind= 0.0.0.0:8333
```